README.md

```diff
@@ -95,6 +85,9 @@ gaps.
 ### Up Next
 
 - Investigate [Node.js inspector](https://nodejs.org/en/learn/getting-started/debugging)
+- Allow long if with then instrs on first line
```

Comment:
The ADR calls out why I may not want to do this. I'd say remove this from TODOs.